Documentation/contributing/development/dev_setup.rst

```diff
@@ -111,6 +111,9 @@ brought up by vagrant:
 * ``INSTALL=1``: Restarts the installation of Cilium, Kubernetes, etc. Only
   useful when the installation was interrupted.
 * ``MAKECLEAN=1``: Execute ``make clean`` before building cilium in the VM.
+* ``NO_BUILD=1``: Does not run the "build" provision step in the VM. Assumes
+  the developer had previously executed ``make build`` before provisioning the
+  VM.
```

Comment:
What's the difference between `NO_PROVISION and `NO_BUILD`? I've been using `NO_PROVISION` specifically for its ability to avoid building.

Reply:
it looks to me that `NO_PROVISION` does not run `sudo make install` inside the VMs?